Modules/III-k6-Intermediate/08-Setting-load-profiles-with-executors.md

```diff
@@ -15,9 +15,9 @@ _Shared Iterations_ is the most basic of the executors. As can be inferred from
 | `maxDuration` | Forcibly stop your test if not finished within this timeframe | `"10m"` |
 | `vus`         | Number of virtual users to run concurrently                   | `1`     |
 
-As noted above, the primary objective is to perform your test `iterations` number of times, over a time period not to exceed `maxDuration`. At any time during the test scenario, there should be `vus` iteration(s) happening unless the total desired iterations has been reached. In this scenario, it is possible for some VUs to perform more work than others.
+As noted above, the primary objective is to perform your test `iterations` number of times, over a time period not to exceed `maxDuration`. At any time during the test scenario, there should be `vus` iteration(s) happening unless the total desired iterations have been reached. In this scenario, it is possible for some VUs to perform more work than others.
```

Comment:
```suggestion
As noted previously, the primary objective is to perform your test `iterations` number of times, over a time period not to exceed `maxDuration`. At any time during the test scenario, there should be `vus` iteration(s) happening unless the total desired iterations have been reached. In this scenario, it is possible for some VUs to perform more work than others.
```

Reply:
Generally, we should avoid directional language for better accessibility:

> Don't use directional language to orient the reader, such as above, below, or right-hand side. This type of language doesn't work well for accessibility or for localization reasons.

https://developers.google.com/style/accessibility

Reply:
(I see this is outside of the scope of this PR's edits, but it was on  the same line :-) )